src/chargemaster_parsers/parsers/cedars_sinai.py

```diff
@@ -0,0 +1,67 @@
+from .parsers import ChargeMasterEntry
+import openpyxl
+
+
+class CedarsSinaiChargeMasterParser:
+    INSTITUTION_NAME = "Cedars-Sinai"
+    ARTIFACT_URL = "https://www.cedars-sinai.org/content/dam/cedars-sinai/billing-insurance/documents/cedars-sinai-changemaster-july-2022.xlsx"
+    
+    @property
+    def institution_name(self):
+        return CedarsSinaiChargeMasterParser.INSTITUTION_NAME
+    
+    @property
+    def artifact_urls(self):
+        return [CedarsSinaiChargeMasterParser.ARTIFACT_URL]
+    
+    def parse_artifacts(self, artifacts):
+        for artifact_url, artifact in artifacts.items():
+            wb = openpyxl.load_workbook(artifact)
+            charge_code_column = None
+            charge_code_description_column = None
+            cpt_hcpcs_code_column = None
+            op_charge_column = None
+            ip_charge_column = None
+
+            for _, row in enumerate(wb.worksheets[0].iter_rows(min_row=5)):
```

Comment:
You don't need to enumerate if you're going to throw away the index parameter.
```suggestion
            for row in wb.worksheets[0].iter_rows(min_row=5):
```
The min_row flag is pretty cool though, I didn't see that when I glanced at the docs.

Reply:
fixed